config/logstash.yml

```diff
@@ -50,6 +50,15 @@
 #
 # pipeline.unsafe_shutdown: false
 #
+# Defaults to false. Setting this to true will rescue all unexpected errors in the
```

Comment:
"rescue" still seems the wrong term. If I put my user hat on, "rescue" feels like a "save the event!" kind of thing, when really we aren't saving or rescuign it, we're throwing it away... it's more like cutting out a bad thing...

Maybe we can combine the 2nd and 3rd sentences "If set to true, Logstash will take the following action on an event if it causes unexpected errors during processing: ... [explanation here]. If set to false, ..."